### DISCUSSION TITLE
TermuxArch version 2.1.882

### DISCUSSION THREAD

**Discussion by SDRausty** [TermuxArch/TermuxArch#98] [General]
TermuxArch version 2.1.882 setup shell script will attempt to set Arch Linux up in your Termux bash environment.  Please see `setupTermuxArch h[elp]` for options how to run [`setupTermuxArch`](https://raw.githubusercontent.com/TermuxArch/TermuxArch/master/setupTermuxArch) on device.  You can use `bash setupTermuxArch` to install Arch Linux in a [Termux](https://github.com/termux/) [PRoot](https://github.com/termux/proot) container on your Android, Chromebook, Fire OS and Windows smartphone, tablet and wearable.  When successfully completed, you will be experiencing the pleasure of the Linux command prompt in Arch Linux in Termux PRoot on smartphone, tablet and wearable.  
                                  
#### Screenshot from an Android smartphone 2018                                    [![screenshot_20180413-172406](https://user-images.githubusercontent.com/27742457/38758637-ec0ff0dc-3f3f-11e8-802c-82bc511cde88.png)](https://termuxarch.github.io/TermuxArch/docs/install) 

* Comments are welcome at [https://github.com/TermuxArch/TermuxArch/issues](https://github.com/TermuxArch/TermuxArch/issues?q=is%3Aissue+is%3Aclosed) ✍                                                                                                      
* Pull requests are welcome [https://github.com/TermuxArch/TermuxArch/pulls](https://github.com/TermuxArch/TermuxArch/pulls?q=is%3Apr+is%3Aclosed) ✍

<hr /><em>This discussion was created from the release <a href='https://github.com/TermuxArch/TermuxArch/releases/tag/2.1.882'>TermuxArch version 2.1.882</a>.</em>

**Comment by raccoonsteven** [comment]
WTF! are you doing?
